### DISCUSSION TITLE
Error in ST output: "requests.exceptions.ChunkedEncodingError"

### DISCUSSION THREAD

**Discussion by hydroculator** [schemathesis/schemathesis#1953] [Q&A] [answered]
I am seeing this message about 30,000 times in the output of the ST run:

requests.exceptions.ChunkedEncodingError(urllib3.exceptions.ProtocolError("Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", urllib3.exceptions.InvalidChunkLength(<urllib3.response.HTTPResponse object at 0x7fec184a7490>, b''))),

Is this a ST error, or is this an API issue? Is there a way I can diagnose this? There isn't much to go on here.

**Comment by Stranger6667** [comment, marked as answer]
Hi @hydroculator 

Newer Schemathesis versions report such problems more concisely. The underlying issue is that the server declared chunked encoding but sent an invalid chunk - I've seen this reported previously with Go -based app, but don't have more info :(
I'd suspect that the server implementation does not handle all possible requests well, and will blame the framework / webserver (e.g. some servers do not support all byte ranges for headers that are specified in the current RFCs)

**Reply by hydroculator** [reply]
Thanks for the help, I'll look into this.
